### PR TITLE
Initial pass at a FAQ

### PR DESCRIPTION
**Pull request by CoralineAda**
Based on a suggestion by @CristinaMendonca in issue #575, this is a first draft of an FAQ for the Contributor Covenant web site. Thoughtful comments are welcome and encouraged.

### CODE REVIEW

**Review by valignatev — COMMENTED** [review]
https://github.com/ContributorCovenant/contributor_covenant/pull/596/files#diff-abe1093d503b2599a2874aa73d7be09cR23

Maybe point to a couple of such studies? Without an ambition to settle it once and for all, but at least to point people to a starting direction from which they could learn further about the topic.

Other than that and a couple mixed second and third level headers, it's a quite eloquent list

**Comment by potato20** [comment]
```Studies show that the two most important factors in successful teams are diversity and psychological safety.```

What kind of diversity? Having multi-disciplinary team members? Or multi-ethnic? If so, where are the studies?

File: content/faq.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### What should I do if I have been accused of violating the code of conduct?
+The first step when dealing with accusations of violation of the code of conduct is to deal with it in a professional manner. Do not respond with sarcasm, attack the code of conduct or its enforcement, or the accuser. Calmly focus on the evidence of what happened, work to understand the damage you may have caused, and examine how your intentions may have inadvertently caused harm. The more you stick to the facts of what happened, and demonstrate your willingness to work with the administrators evaluate the claims, the better. Most importantly, trust the project maintainers to act in a fair and just manner.
+
+### Won't a code of conduct have a negative effect on the end product?
+There is no evidence that adopting a code of conduct has a negative effect on code quality. But it will affect the experience of those that participate in communities where the CoC is enforced. Ensuring that more people feel safe contributing to the project can significantly improve the quality of the codebase.
+
+## Won't this just promote participation for the sake of participation and an end of meritocracy?
```
Comment (seojangho) [inline content/faq.md]: Maybe `###` should be used instead of `##`? Same comments for the following sections.

**Comment by CameronNemo** [comment]
Can you clarify the meaning of inclusive language.

**Pull request merged**